.changeset/sharp-nails-clean.md

```diff
@@ -0,0 +1,52 @@
+---
+'@graphql-cli/codegen': major
```

Comment:
Do we really need a major release?

Reply:
the changesets can be changed, I think this change is for minor, it was set as major since at first there was some changes that I thought were breaking, now everything should be fine

Reply:
@ardatan done 👍